datastore/tests/system/test_system.py

```diff
@@ -442,6 +443,25 @@ def test_transaction_via_with_statement(self):
         self.case_entities_to_delete.append(retrieved_entity)
         self.assertEqual(retrieved_entity, entity)
 
+    def test_readonly_put(self):
```

Comment:
@chemelnucfin This **should not** be a system test. It is testing behavior of the client that **does not** depend on a live network.

Reply:
@dhermes how is this different than the test above it?

Reply:
The test above it (`test_transaction_via_with_statement`) actually talks to the network.